### PR TITLE
Zib-BowelFunction

### PR DESCRIPTION
**Pull request by Ashwin-nictiz**

### CODE REVIEW

File: resources/zib/zib-MedicalDevice.xml [hunk 1]
```diff
@@ -206,10 +211,12 @@
       <comment value="There are several zibs that reference zib MedicalDevice on a functional level, but for which the direction of the reference in FHIR is reversed. Consult the mappings on this element for an overview of which zib concepts are mapped to this element. &#xD;&#xA;&#xD;&#xA;Note that some zibs have a reference to MedicalDevice that is further constrained (e.g. with additional constraints on ProductType or AnatomicalLocation), such as HearingFunction and VisualFunction. For these zibs separate DeviceUseStatement profiles have been created (based on this profile) which also include a specific slice on which their respective MedicalDevice concept mapping is present." />
       <alias value="FunctioneleOfMentaleStatus" />
       <alias value="Mobiliteit" />
+      <alias value="Darmfunctie" />
       <type>
         <code value="Reference" />
         <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-FunctionalOrMentalStatus" />
         <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-Mobility" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-BowelFunction" />
```
Comment (LuudSlagter) [inline resources/zib/zib-MedicalDevice.xml:219]: The nl-core variant of this targetProfile should be added to nl-core-MedicalDevice.

File: resources/nl-core/nl-core-BowelFunction.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  <purpose value="A derived profile from [zib-BowelFunction](http://nictiz.nl/fhir/StructureDefinition/zib-BowelFunction) to provide a version better suited for implementation purposes. This profile augments the base profile with elements found in the various use cases that have adopted the zib." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <kind value="resource" />
+  <abstract value="false" />
+  <type value="Observation" />
+  <baseDefinition value="http://nictiz.nl/fhir/StructureDefinition/zib-BowelFunction" />
+  <derivation value="constraint" />
+  <differential>
```
Comment (LuudSlagter) [inline resources/nl-core/nl-core-BowelFunction.xml:26]: The reference to nl-core-Patient on `.subject` is missing.

File: resources/zib/zib-BowelFunction.xml [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+      <sliceName value="stoma" />
+      <min value="0" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-StomaReference" />
+      </type>
+      <isModifier value="false" />
+    </element>
+    <element id="Observation.extension:stoma.value[x]">
```
Comment (LuudSlagter) [inline resources/zib/zib-BowelFunction.xml:65]: In the nl-core profile, this reference should be changed to nl-core-Stoma.

File: resources/zib/zib-BowelFunction.xml [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Observation" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Observation" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Observation">
+      <path value="Observation" />
+      <short value="BowelFunction" />
+      <comment value="Please note that on a functional level, zib BowelFunction references zib MedicalDevice, but in FHIR this direction is reversed. Therefore, the concept MedicalDevice (NL-CM:4.15.6) is mapped on `DeviceUseStatement.reasonReference` in profile [zib-MedicalDevice](http://nictiz.nl/fhir/StructureDefinition/zib-MedicalDevice) instead of in this profile." />
```
Comment (LuudSlagter) [inline resources/zib/zib-BowelFunction.xml:35]: The concept NL-CM:4.15.6 is called IncontinenceMaterial, not MedicalDevice.

Comment (LuudSlagter) [reply resources/zib/zib-BowelFunction.xml:35]: Moreover in the corresponding text in the nl-core profile, 'zib-MedicalDevice' should be changed to 'nl-core-MedicalDevice'.

**Review by LuudSlagter — CHANGES_REQUESTED** [review]

File: resources/nl-core/nl-core-BowelFunction.xml (hunk 2 above)
Comment (Ashwin-nictiz) [reply resources/nl-core/nl-core-BowelFunction.xml:26]: Fix

File: resources/zib/zib-BowelFunction.xml (hunk 3 above)
Comment (Ashwin-nictiz) [reply resources/zib/zib-BowelFunction.xml:64]: Fix

File: resources/zib/zib-MedicalDevice.xml (hunk 1 above)
Comment (Ashwin-nictiz) [reply resources/zib/zib-MedicalDevice.xml]: Done

File: resources/zib/zib-BowelFunction.xml (hunk 4 above)
Comment (Ashwin-nictiz) [reply resources/zib/zib-BowelFunction.xml]: Fix

File: examples/nl-core-BowelFunction-02-DeviceUseStatement-01.xml [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+   <device>
+      <reference value="Device/nl-core-BowelFunction-02-Device-01"/>
+      <type value="Device"/>
+      <display value="Elastisch broekje"/>
+   </device>
+   <reasonReference>
+      <reference value="Observation/nl-core-BowelFunction-02"/>
+      <type value="Observation"/>
+      <display value="Bowel function observation, fecal continence: incidenteel ontlastingsverlies, defecation consistency: Zachte stukjes met gehavende randen"/>
+   </reasonReference>
```
Comment (Ashwin-nictiz) [inline examples/nl-core-BowelFunction-02-DeviceUseStatement-01.xml:22]: The validator gives the following error:

```
  -  error at DeviceUseStatement.reasonReference[0] (18, 21):
     Slicing cannot be evaluated: Problem evaluating slicing expression for element in profile http://nictiz.nl/fhir/StructureDefinition/nl-core-MedicalDevice path DeviceUseStatement.reasonReference[0] (fhirPath = true and (resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-BowelFunction') or resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-FunctionalOrMentalStatus') or resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-Mobility') or resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-Stoma'))): The URL 'Observation/nl-core-BowelFunction-02' is not known to the FHIR validator, and has not been provided as part of the setup / parameters
  -  error at DeviceUseStatement.reasonReference[0] (18, 21):
     Slicing cannot be evaluated: Problem evaluating slicing expression for element in profile http://nictiz.nl/fhir/StructureDefinition/nl-core-MedicalDevice path DeviceUseStatement.reasonReference[0] (fhirPath = true and resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-Problem')): The URL 'Observation/nl-core-BowelFunction-02' is not known to the FHIR validator, and has not been provided as part of the setup / parameters
```

File: examples/nl-core-BowelFunction-01.xml [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+            <display value="Als een worst of slang, glad en zacht"/>
+         </coding>
+      </valueCodeableConcept>
+   </component>
+   <component>
+      <code>
+         <coding>
+            <system value="http://loinc.org"/>
+            <code value="9397-1"/>
+            <display value="Color of Stool"/>
```
Comment (Ashwin-nictiz) [inline examples/nl-core-BowelFunction-01.xml:63]: There is a Dutch translation for this code:

"Kleur"

File: examples/nl-core-BowelFunction-02.xml [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+            <display value="Zachte stukjes met gehavende randen"/>
+         </coding>
+      </valueCodeableConcept>
+   </component>
+   <component>
+      <code>
+         <coding>
+            <system value="http://loinc.org"/>
+            <code value="9397-1"/>
+            <display value="Color of Stool"/>
```
Comment (Ashwin-nictiz) [inline examples/nl-core-BowelFunction-02.xml:75]: There is a Dutch translation for this code:

"Kleur"

File: examples/nl-core-BowelFunction-02-DeviceUseStatement-01.xml (hunk 5 above)
Comment (LuudSlagter) [reply examples/nl-core-BowelFunction-02-DeviceUseStatement-01.xml:22]: Which validator did you use? In this PR these errors are not present.

Comment (Ashwin-nictiz) [reply examples/nl-core-BowelFunction-02-DeviceUseStatement-01.xml:22]: QA tool in the repo and simplifier.

**Pull request closed without merging**